Repo.rb

```diff
@@ -20,9 +20,9 @@ def initialize(dir, subrepos, is_head = false)
   #class initializers
 
   # Initializes/Re-initializes an Rgit repo in a directory
-  def self.init(dir, options = {})
+  def self.init(dir, opt_str)
     # 1. Initialize a git repository in the directory
-    Git.init(dir, options)
+    output = git_command('init', opt_str, dir)
```

Comment:
Useless assignment to variable - `output`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.